package/yast2-network.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Mon Apr 23 18:10:29 UTC 2018 - mfilka@suse.com
+
+- bnc#1089471 
+  - fixed option check to get proper proposal
```

Comment:
The changelog could be more descriptive about the problem and solution, read it alone without context...